core/src/main/java/de/danoeh/antennapod/core/service/download/DownloadService.java

```diff
@@ -225,6 +241,7 @@ public void onDestroy() {
             downloadPostFuture.cancel(true);
         }
         unregisterReceiver(cancelDownloadReceiver);
+        unRegisterLollipopNetworkListener();
```

Comment:
Should this have a API version guard as well? Alternatively, how about moving the guard inside the connection monitor?